lib/driver/runtimeTool.cpp

```diff
@@ -101,7 +101,24 @@ ToolOnModule(WasmEdge::VM::VM &VM, const std::string &FuncName,
       case TypeCode::V128:
         fmt::print("{}\n"sv, uint128((*Result)[I].first.get<uint128_t>()));
         break;
-      /// TODO: FuncRef and ExternRef
+      case TypeCode::Ref: {
+        if((*Result)[I].second.isFuncRefType())
```

Comment:
Please follow the coding style, add the `{}` for the `if` statements.
And also please pass the clang-format.